src/assets/style/main.scss

```diff
@@ -18,3 +18,12 @@ $color-green:#008000;
 #password-err {
   text-align: left
 }
+
+.center_password_form{
```

Comment:
The css classes do not follow the BEM naming conventions. Please refactor your css names.

Reply:
Thank you @bettblake08. The feedback has been implemented.